### DISCUSSION TITLE
future plans?

### DISCUSSION THREAD

**Discussion by KaiBVG** [flybywiresim/aircraft#3386] [General]
Good morning,
I have to say a big "Thanks!" for your outstanding work! I'm coming from the P3D-platform and such a development would be impossible there.
Just some small questions regarding your further development: is it planed building in vertical navigation things like the deviation pointer beside the altitude tape or exactly given FL's beside a waypoint in the CDU? Even a second autopilot for autolanding? I'm looking very forward to these things!

**Comment by hotshotp** [comment]
If it exists on a real a320, then of course we will implement it. The things you listed are core features of the flight plan system in the airbus. These are for sure coming and are in development, so it's just a matter of time, since they are hard to develop!

**Reply by KaiBVG** [reply]
Wow, that sounds very good! Please keep up your hard good work!
